frontend/public/components/cluster-overview.jsx

```diff
@@ -8,12 +8,12 @@ import { NavTitle, AsyncComponent, Firehose, StatusBox, tectonicHelpBase, OpenSh
 import { k8sBasePath } from '../module/k8s';
 import { SecurityScanningOverview } from './secscan/security-scan-overview';
 import { StartGuide } from './start-guide';
-import { Gauge, prometheusBasePath } from './graphs';
+import { Gauge, prometheusBasePath, requirePrometheus } from './graphs';
 import { Status, errorStatus } from './graphs/status';
 import { EventStreamPage } from './events';
 import { SoftwareDetails } from './software-details';
 import { formatNamespacedRouteForResource } from '../ui/ui-actions';
-import { FLAGS, connectToFlags } from '../features';
+import { connectToFlags, FLAGS } from '../features';
```

Comment:
Not sure if this change is needed

Reply:
Yeah, that was unintentional (although I do prefer to alphabetize imports)